torch/_inductor/runtime/halide_helpers.py

```diff
@@ -0,0 +1,168 @@
+# mypy: allow-untyped-defs
+import halide as hl  # type: ignore[import-untyped, import-not-found]
```

Comment:
I think we will need to do something like:
```suggestion
try:
  import halide as hl  # type: ignore[import-untyped, import-not-found]
except ImportError:
  hl = None
```

This file shouldn't throw exceptions when imported without Halide installed.

Reply:
triton_helper doesn't guard the import. I can do it to be safe.